intro/scipy.rst

```diff
@@ -448,9 +448,7 @@ our initial guess: ::
 
 Suppose we have data sampled from ``f`` with some noise: ::
 
-.. Comment to make doctest pass
```

Comment:
I'd like to keep this code hidden in a comment. Indeed it was breaking indentation, and thus needs to be moved elsewhere.